docs/versions/unversioned/sdk/barometer.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Barometer
+---
+
+Access the device barometer sensor to respond to changes in air pressure. Currently data is passed back as raw values. You will need to convert the units yourself. Below we've included information regarding the exact format data is returned in.
+
+| OS      | Units               | Provider                                                                                                | Description                                                                                                                         |
+| ------- | ------------------- | ------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| iOS     | kilopascal          | [`CMAltimeter`](https://developer.apple.com/documentation/coremotion/cmaltimeter)                       | Altitude events reflect the change in the current altitude, not the absolute altitude.                                              |
```

Comment:
Would it make sense to multiply the result by 10 on iOS to have the API return the pressure as hPa on both platforms? Having different units on different platforms seems annoying.